src/style.scss

```diff
@@ -0,0 +1,144 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.container {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid white;
+  background-color: white;
+  text-align: center;
+  overflow: hidden;
+
+  [data-qa="card"] {
```

Comment:
Consider using class selectors for styling

Reply:
Not fixed